src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const VERY_SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
+  const OVER_SHORT_TERM = 6;
+  const PRE_SHORT_TERM = 3;
+  const SHORT_TERM_DISCAUNT = 20;
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCAUNT;
+  }
+
+  if (days <= VERY_SHORT_TERM) {
+    return PRICE_PER_DAY * VERY_SHORT_TERM;
```

Comment:
The logic here is incorrect. You should return the 'basePrice' for days less than or equal to 2, not 'PRICE_PER_DAY * VERY_SHORT_TERM'. This will always return 80, which is incorrect for 1 day rental.